### PR TITLE
Use upstream cvd binary.

### PR DESCRIPTION
**Pull request by ser-io**

### CODE REVIEW

File: frontend/src/host_orchestrator/orchestrator/createcvdaction_test.go [hunk 1]
```diff
@@ -170,7 +158,7 @@ func TestCreateCVDVerifyStartCVDCmdArgs(t *testing.T) {
 	dir := orchtesting.TempDir(t)
 	defer orchtesting.RemoveDir(t, dir)
 	goldenPrefixFmt := fmt.Sprintf("sudo -u fakecvduser "+
-		"ANDROID_HOST_OUT=%[1]s/artifacts/%%[1]s "+"%[1]s/cvd --group_name=cvd start --daemon --report_anonymous_usage_stats=y"+
+		"ANDROID_HOST_OUT=%[1]s/artifacts/%%[1]s "+"/usr/bin/cvd --group_name=cvd start --daemon --report_anonymous_usage_stats=y"+
```
Comment (jemoreira) [inline frontend/src/host_orchestrator/orchestrator/createcvdaction_test.go:161]: Could this use `cvdBin`?

File: frontend/src/host_orchestrator/orchestrator/createcvdaction.go [hunk 2]
```diff
@@ -167,11 +157,11 @@ func (a *CreateCVDAction) launchWithCanonicalConfig(op apiv1.Operation) (*apiv1.
 	opts := cvd.CommandOpts{
 		Timeout: a.cvdStartTimeout,
 	}
-	cmd := cvd.NewCommand(a.execContext, a.paths.CVDBin(), args, opts)
+	cmd := cvd.NewCommand(a.execContext, cvdBin, args, opts)
```
Comment (jemoreira) [inline frontend/src/host_orchestrator/orchestrator/createcvdaction.go:160]: nit: With several files in this package it's not immediately obvious where `cvdBin` is declared and whether it's a constant or global variable. Consider moving all constants to a constants.go file or use `cvdBin` only from the file it's declared in.

**Review by jemoreira — APPROVED** [review]

File: frontend/src/host_orchestrator/orchestrator/createcvdaction_test.go (hunk 1 above)
Comment (ser-io) [reply frontend/src/host_orchestrator/orchestrator/createcvdaction_test.go]: done

File: frontend/src/host_orchestrator/orchestrator/createcvdaction.go (hunk 2 above)
Comment (ser-io) [reply frontend/src/host_orchestrator/orchestrator/createcvdaction.go]: moved the constants to the `cvd` package

**Pull request merged**